powerscale/provider/snapshot_datasource.go

```diff
@@ -158,6 +158,46 @@ func (d *SnapshotDataSource) Schema(ctx context.Context, req datasource.SchemaRe
 					"path": schema.StringAttribute{
 						Optional: true,
 					},
+
+					"name": schema.StringAttribute{
```

Comment:
path and name should be mutually exclusive right? what if user tries to give both?
Also, these should be mutually exclusive with all other filters like sort, limit and so on